### PR TITLE
Added news story queries test and other misc updates

### PR DESCRIPTION
**Pull request by penny-lischer**
Removed Paragraph.tsx and Paragraph.test.tsx
Added additional tests to BenefitHub

## Description
-Added news story queries test.
-Removed Paragraph.tsx and Paragraph.test.tsx
-Added additional tests to BenefitHub-

### CODE REVIEW

File: src/pages/[...slug].tsx [hunk 1]
```diff
@@ -50,6 +50,8 @@ export async function getStaticProps(context) {
     }
   }
 
+  console.log('context ', context)
+  console.log('path.entity.uuid ', path.entity.uuid)
```
Comment (amponce) [inline src/pages/[...slug].tsx:54]: 🌴

File: src/pages/[...slug].tsx [hunk 2]
```diff
@@ -50,6 +50,8 @@ export async function getStaticProps(context) {
     }
   }
 
+  console.log('context ', context)
```
Comment (timcosgrove) [inline src/pages/[...slug].tsx:53]: Let's pull these out

File: src/data/queries/tests/mocks/newsStory.mock.json [hunk 3]
```diff
@@ -0,0 +1,295 @@
+{
```
Comment (timcosgrove) [inline src/data/queries/tests/mocks/newsStory.mock.json:1]: Let's move this into `src/mocks`

**Review by timcosgrove — APPROVED** [review]
Approved after changes

**Pull request merged**